CMakeLists.txt

```diff
@@ -286,7 +286,7 @@ ELSEIF(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
             BUNDLE DESTINATION .)
 ELSEIF(CMAKE_SYSTEM_NAME STREQUAL "Windows")
     INSTALL(PROGRAMS
-            "${CMAKE_SOURCE_DIR}/bin/${PROJECT_TARGET_BIN}.exe"
+            "${CMAKE_SOURCE_DIR}/bin/Release/${PROJECT_TARGET_BIN}.exe"
```

Comment:
Under VS build multi-tagret generator is used, so build type is unknown at configure time, so we cannot use hardcoded paths here - we must replace `INSTALL(PROGRAMS` with `INSTALL(TARGETS` here...